kodilla-exception/src/main/java/com/kodilla/exception/test/FlightFinder.java

```diff
@@ -0,0 +1,27 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class FlightFinder {
+
+    public String findFlight(Flight flight) throws RouteNotFoundException {
+        Map<String,Boolean> arrivalAirports = new HashMap<>();
+        arrivalAirports.put("Warsaw", true);
+        arrivalAirports.put("Berlin", true);
+        arrivalAirports.put("London", false);
+
+        Optional<Boolean> arrivalAirportExist = Optional.ofNullable(arrivalAirports.get(flight.getArrivalAirport()));
```

Comment:
Nie, jesli nie ma w mapie wpisu -> rzucasz wyjatek.
Jak jest w mapie wpis, to jesli jest true to zwracasz Optional.of(flight) w przeciwnym razie zwracasz Optional.empty()

Reply:
Nie rozumiem tego co napisałeś. Jakieś wskazówki?